docs/user/reference/juju-cli/list-of-juju-cli-commands/info.md

```diff
@@ -11,14 +11,21 @@ Displays detailed information about CharmHub charms.
 ### Options
 | Flag | Default | Usage |
 | --- | --- | --- |
-| `--arch` | all | specify an arch &lt;all&#x7c;amd64&#x7c;arm64&#x7c;ppc64el&#x7c;riscv64&#x7c;s390x&gt; |
-| `--base` |  | specify a base |
-| `--channel` |  | specify a channel to use instead of the default release |
+| `--arch` | all | Specify an arch &lt;all&#x7c;amd64&#x7c;arm64&#x7c;ppc64el&#x7c;riscv64&#x7c;s390x&gt; |
+| `--base` |  | Specify a base |
+| `--channel` |  | Specify a channel to use instead of the default release |
 | `--charmhub-url` | https://api.charmhub.io | specify the Charmhub URL for querying the store |
-| `--config` | false | display config for this charm |
+| `--config` | false | Display config for this charm |
 | `--format` | tabular | Specify output format (json&#x7c;tabular&#x7c;yaml) |
 | `-o`, `--output` |  | Specify an output file |
+<<<<<<< HEAD
 | `--unicode` | auto | display output using unicode &lt;auto&#x7c;never&#x7c;always&gt; |
+=======
```

Comment:
One more conflict here, thought to fix this one just do:
```
make juju
make docs-html
```
That'll regenerate these docs.

Reply:
Fixed.